lib/mux/video/assets.ex

```diff
@@ -120,7 +120,7 @@ defmodule Mux.Video.Assets do
       #{inspect(Fixtures.asset())}
 
   """
-  def update_master_access(client, asset_id, params \\ []) do
```

Comment:
Technically, an API did change, does that mean this needs to be a `1.9` release instead of `1.8.1` 😭 

I don't *think* so, because If you did call `update_mp4_support/2` or `update_master_access/2` you would get an error from the API. And I literally released `1.8` like 10 minutes ago so there's no way anyone is using it yet.

Reply:
yeah haha, let's keep this a minor version bump.